WISE-Community/WISE-Client
# refactor(Branch): Separate create and edit branch paths

## Changes
- Rename some components so we now have
  - CreateBranchComponent
  - CreateBranchPathsComponent
  - EditBranchComponent
  - EditBranchPathsComponent
- Make EditBranchPathsComponent extend CreateBranchPathsComponent

## Test
- Make sure creating a branch still works
- Make sure editing a branch still works
